client/v2/autocli/keyring/no_keyring.go

```diff
@@ -0,0 +1,11 @@
+package keyring
+
+import "errors"
+
+var _ Keyring = NoKeyring{}
+
+type NoKeyring struct{}
```

Comment:
is this for testing purposes?

Reply:
No, it is a fallback if you do not provide any keyring (where client/v2 will keep the same behavior as today).